swiftinterfaceparser/Makefile

```diff
@@ -0,0 +1,38 @@
+
+ANTLRJAR=antlr/antlr-4.9.1-complete.jar
+ANTLR=java -jar $(ANTLRJAR)
+CLASSPATH=.:$(ANTLRJAR)
+
+GRAMMAR=SwiftInterface.g4
+
+CSOUTDIR=CSParser
+JAVAOUTDIR=JavaParser
+
+JDK_VERSION=$(shell ls /usr/local/Cellar/openjdk | sort -V | tail -n 1)
+JDK_PATH="/usr/local/Cellar/openjdk/$(JDK_VERSION)"
+JAVA_HOME=$(JDK_PATH)
```

Comment:
These variables don't seem to be used.

Reply:
`JAVA_HOME` is used by `java` and `java`